engine/executor/dm/unitholder.go

```diff
@@ -269,7 +274,11 @@ func (u *unitHolderImpl) CheckAndUpdateStatus(ctx context.Context) {
 	sourceStatus := u.getSourceStatus()
 
 	if sourceStatus == nil || time.Since(sourceStatus.UpdateTime) > sourceStatusRefreshInterval {
-		u.updateSourceStatus(ctx)
+		u.bgWg.Add(1)
+		go func() {
+			defer u.bgWg.Done()
+			u.updateSourceStatus(ctx)
+		}()
```

Comment:
Will it leave too many goroutines for a long time(depending on the timeout of `updateSourceStatus`) if `updateSourceStatus` is quite slow, since the `Tick` is called quite frequently.

Reply:
45ba210